websites/Y/YouTube/presence.ts

```diff
@@ -331,7 +331,8 @@ presence.on("UpdateData", async () => {
 					!document.querySelector("#text.ytd-channel-name")?.textContent &&
 					documentTitle.includes(
 						document.querySelector("#text.ytd-channel-name")?.textContent
-					)
+					) &&
+					document.querySelector("#text.ytd-channel-name")?.textContent
```

Comment:
It gets ! checked and then !! checked?

Reply:
Confused but this somehow fixes it?

Reply:
well one of these checks seems useless...